src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -12,6 +12,10 @@
 
 namespace py = pybind11;
 
+void regclass_RemoteTensor(py::module m) {
+    py::class_<ov::RemoteTensor, ov::Tensor, std::shared_ptr<ov::RemoteTensor>> cls(m, "RemoteTensor");
```

Comment:
Also I think it should be exposed on `__init__.py` levels.

Moreover, `RemoteTensor` is not clear here. Is it just a Tensor? If so, what is the goal of introducing it?
I assume `RemoteTensor` API is not 1:1 copy and it is way different in behavior than Tensor. Especially considering different ways of obtaining underlaying memory.

If simple inheritance is applied it will be way harder to deprecate parts of Remote version later. Is there any planned design for the future?